trl/trainer/grpo_trainer.py

```diff
@@ -359,6 +366,10 @@ def __init__(
                 reward_funcs[i] = AutoModelForSequenceClassification.from_pretrained(
                     reward_func, num_labels=1, **model_init_kwargs
                 )
+                if args.disable_dropout:
+                    if isinstance(reward_funcs[i], nn.Module):
```

Comment:
I think `AutoModelForSequenceClassification` is loaded in eval model by default, so technically we don't need this here (happy to keep it though if we want to be safe)